server/memphis_handlers_schemas.go

```diff
@@ -451,20 +439,18 @@ func (sh SchemasHandler) RemoveSchema(c *gin.Context) {
 		}
 		if exist {
 			DeleteTagsBySchema(schema.ID)
-
+			var schemaIds []primitive.ObjectID
+			schemaIds = append(schemaIds, schema.ID)
+
+			err := sh.findAndDeleteSchema(schemaIds)
```

Comment:
you are inside for loop this should happen outside the for loop same for the log